pkg/api/types.go

```diff
@@ -1390,7 +1401,7 @@ func (p ClusterProfile) LeaseType() string {
 // LeaseTypeFromClusterType maps cluster types to lease types
 func LeaseTypeFromClusterType(t string) (string, error) {
 	switch t {
-	case "aws", "aws-arm64", "aws-c2s", "aws-china", "aws-usgov", "alibaba", "azure-2", "azure4", "azure-arc", "azurestack", "azuremag", "equinix-ocp-metal", "gcp", "libvirt-ppc64le", "libvirt-s390x", "openstack", "openstack-osuosl", "openstack-vexxhost", "openstack-ppc64le", "vsphere", "ovirt", "packet", "packet-edge", "kubevirt", "aws-cpaas", "osd-ephemeral":
+	case "aws", "aws-arm64", "aws-c2s", "aws-china", "aws-usgov", "aws-sc2s", "alibaba", "azure-2", "azure4", "azure-arc", "azurestack", "azuremag", "equinix-ocp-metal", "gcp", "libvirt-ppc64le", "libvirt-s390x", "openstack", "openstack-osuosl", "openstack-vexxhost", "openstack-ppc64le", "vsphere", "ovirt", "packet", "packet-edge", "kubevirt", "aws-cpaas", "osd-ephemeral":
```

Comment:
Missing `aws-qe`?

Reply:
@petr-muller `aws-qe` is the cluster profile of QE, the cluster type is the same `aws` profile which is  **aws**

Reply:
Does `aws-qe` have separate leases?

Reply:
yes, it does, https://github.com/openshift/release/pull/27033/files#diff-aa86f662b34188da9ad50af7c05ef19dfa48af3c726cd882949002cf9112eed9R28-R30

Reply:
I thought lease string is handled by [1] via cluster profile name, is `aws-qe` required here? but it's not a _cluster type_ , I'm a little confused

[1] https://github.com/openshift/ci-tools/pull/2724/files#diff-dfee74d02fd350fb1ee1ffbbbc32acf4cb075c9ae42b6e31c8c7acefe7fed623R1306-R1315

Reply:
@petr-muller is there anything that I need to update before getting approval?

Reply:
STOP PINGING ME

Reply:
I review code in batches at my convenience. Stop triggering a million notifications for me. I am aware of the PR and will respond. I will not respond on-demand.

Reply:
I apologize for disturbing you with my message, thanks for reviewing my PRs.